client/dist/style.css

```diff
@@ -0,0 +1,17 @@
+.review-single-star-outline {
+  height: 36px;
+  width: 31px;
+}
+
+.review-single-star-fill {
+  position: relative;
+  display: inline-block;
+  height: 36px;
+  background-color: #333333;
+}
+
+.review-single-star-container {
+  height: 36px;
+  width: 31px;
+  display: inline-block;
+}
```

Comment:
Please put all of your components styling into the subCSS folder with the file named ratings.css